exchange/bitswap/wantmanager.go

```diff
@@ -96,7 +96,7 @@ func (pm *WantManager) SendBlock(ctx context.Context, env *engine.Envelope) {
 	log.Infof("Sending block %s to %s", env.Peer, env.Block)
 	err := pm.network.SendMessage(ctx, env.Peer, msg)
 	if err != nil {
-		log.Noticef("sendblock error: %s", err)
+		log.Warningf("sendblock error: %s", err)
```

Comment:
this should probably just be `log.Infof` -- warnings are things the user may need to be conscious about